### PR TITLE
docs: modify the documentation about "Sheet Management"

### PR DESCRIPTION
**Pull request by graczhual**
- delete the operations on revision
- add description about dict
- add replace sheet

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2gh5pyb Sheet Management 首页重构](https://app.clickup.com/t/2gh5pyb)

File: docs/source/features/sheet_management/index.rst [hunk 1]
```diff
@@ -37,24 +40,60 @@ necessary to get a dataset first:
  Create a Sheet
 ****************
 
-Create a sheet on the open draft:
+SDK supports managing sheets of the open draft. Thus, it is necessary to create or get an open
```
Comment (Lee-000) [inline docs/source/features/sheet_management/index.rst:43]: `of open drafts`?

File: docs/source/features/sheet_management/index.rst [hunk 2]
```diff
@@ -37,24 +40,60 @@ necessary to get a dataset first:
...
+   ...    {"filename": "c.jpg"},
+   ... ]
+   >>> df = DataFrame(data)
+   >>> df
+      filename
+   0  a.jpg
+   1  b.jpg
+   2  c.jpg
+ 
+Or getting the dataframe from other sheets:
```
Comment (Lee-000) [inline docs/source/features/sheet_management/index.rst:67]: Is `from other sheets` appropriate?  Should it be `getting a dataframe by copying other sheets`?

File: docs/source/features/sheet_management/index.rst [hunk 3]
```diff
@@ -37,24 +40,60 @@ necessary to get a dataset first:
...
    draft.upload()
 
 .. note::
    Only changes made on the open draft can be synchronized to Graviti via the method
    :meth:`~graviti.manager.draft.Draft.upload`.
 
-.. important::
-   When setting sheets, if the sheet with the corresponding name already exists, SDK will delete
-   the original one and create a new one.
+Or replacing the old sheet:
```
Comment (Lee-000) [inline docs/source/features/sheet_management/index.rst:91]: Replacing is not supported yet, should we remove the description here?

File: docs/source/features/sheet_management/index.rst [hunk 4]
```diff
@@ -74,57 +113,31 @@ List sheets of an open draft:
...
-
-   tag = dataset.tags.get(f"{TAG_NAME}")
-   list(tag.keys())
-
 *************
  Get a Sheet
 *************
 
-Get the sheet for the specified version of the dataset:
+Get the sheet for the specified version of the dataset by name:
```
Comment (Lee-000) [inline docs/source/features/sheet_management/index.rst:120]: `Get a sheet`?

**Review by Lee-000 — CHANGES_REQUESTED** [review]

File: docs/source/features/sheet_management/index.rst (hunk 3 above)
Comment (graczhual) [reply docs/source/features/sheet_management/index.rst:91]: replace sheet is ok

**Review by Lee-000 — APPROVED** [review]

**Pull request merged**